### PR TITLE
Folder modification

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: apps/examples/tls_client/tls_client_main.c [hunk 1]
```diff
@@ -316,6 +320,8 @@ struct pthread_arg {
 	"    force_ciphersuite=<name>    default: all enabled\n"\
 	" acceptable ciphersuite names:\n"
 
+#define MAX_DATA_SIZE 100*1024*1024 /* Maximum transfer size is 100MB */
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: for coding rule, space is needed between operators

File: apps/examples/tls_client/tls_client_main.c [hunk 2]
```diff
@@ -831,7 +841,13 @@ int tls_client_cb(void *args)
 			if (opt.dhmlen < 0) {
 				goto usage;
 			}
-		} else {
+		} else if( strcmp(p, "aging") == 0) {
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: for coding rule, add space between if and '(' , and remove unnecessary space

File: apps/examples/tls_client/tls_client_main.c [hunk 3]
```diff
@@ -831,7 +841,13 @@ int tls_client_cb(void *args)
...
+			opt.aging = atoi(q);
+			if(opt.aging <= 0 && opt.aging > MAX_DATA_SIZE) {
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: check space after if

File: apps/examples/tls_client/tls_client_main.c [hunk 4]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+		ret = mbedtls_ssl_write(&ssl, (void *)&aging_size, sizeof(unsigned int));
+		if (ret <= 0 && ret != MBEDTLS_ERR_SSL_WANT_READ &&
+			ret != MBEDTLS_ERR_SSL_WANT_WRITE) {
+			mbedtls_printf(" failed to send transfer size to the tls server(%x)\n", -ret);
+			goto exit;
+		}
+		
+		while (send_left > 0) {
+		    int datasize = send_left < MBEDTLS_SSL_MAX_CONTENT_LEN? send_left : MBEDTLS_SSL_MAX_CONTENT_LEN;
+			for( written = 0, frags = 0; written < datasize; written += ret, frags++ )
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: check space

File: apps/examples/tls_client/tls_client_main.c [hunk 5]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+			{
+				int bufsize = datasize - written;
+				while( ( ret = mbedtls_ssl_write( &ssl, buf + written, bufsize ) ) <= 0 )
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: check space

File: apps/examples/tls_client/tls_client_main.c [hunk 6]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+				{
+					if( ret != MBEDTLS_ERR_SSL_WANT_READ &&
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: check space

File: apps/examples/tls_client/tls_client_main.c [hunk 7]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+						ret != MBEDTLS_ERR_SSL_WANT_WRITE )
+					{
+						mbedtls_printf( " failed\n  ! mbedtls_ssl_write returned -0x%x\n\n", -ret );
```
Comment (jeongchanKim) [inline apps/examples/tls_client/tls_client_main.c]: remove unnecessary space before ')'

File: apps/examples/tls_server/tls_server_main.c [hunk 8]
```diff
@@ -367,6 +371,7 @@ struct pthread_arg {
 	"    force_ciphersuite=<name>    default: all enabled\n"            \
 	" acceptable ciphersuite names:\n"
 
+#define MAX_DATA_SIZE 100*1024*1024 // Maximum data size for aging test 100MB
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 9]
```diff
@@ -1238,6 +1244,16 @@ int tls_server_cb(void *args)
 			opt.sni = q;
 		} else if (strcmp(p, "retry") == 0) {
 			opt.retry = atoi(q);
+		} else if (strcmp( p, "aging") == 0){
+			if(strcmp (q, "true") == 0) {
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 10]
```diff
@@ -1238,6 +1244,16 @@ int tls_server_cb(void *args)
 			opt.sni = q;
 		} else if (strcmp(p, "retry") == 0) {
 			opt.retry = atoi(q);
+		} else if (strcmp( p, "aging") == 0){
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: add space between ')' and '{'

File: apps/examples/tls_server/tls_server_main.c [hunk 11]
```diff
@@ -1238,6 +1244,16 @@ int tls_server_cb(void *args)
...
+			if(strcmp (q, "true") == 0) {
+				opt.aging = 1;
+			}
+			else if(strcmp(q,"false") == 0){
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 12]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+	if (opt.aging > 0) {
+		/*
+		 * 6. Read the large data from client
+		 */
+		unsigned char big_buf[1024];
+		int received = 0;
+
+		// read transfer size from the client
+		unsigned int datasize;
+		ret = mbedtls_ssl_read( &ssl, (void *)&datasize, sizeof(unsigned int));
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: remove unnecessary space

File: apps/examples/tls_server/tls_server_main.c [hunk 13]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+
+		if( ret <= 0 &&
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 14]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+		}
+		datasize = ntohl(datasize);
+		mbedtls_printf(" < Read the large data from the client (%d)B\n", datasize);
+
+		struct timeval start_time, end_time;
+		gettimeofday(&start_time, NULL);
+
+		do {
+			ret = mbedtls_ssl_read( &ssl, big_buf, sizeof(big_buf) );
+			if( ret == MBEDTLS_ERR_SSL_WANT_READ ||
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 15]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+			goto exit;
+		}
+		datasize = ntohl(datasize);
+		mbedtls_printf(" < Read the large data from the client (%d)B\n", datasize);
+
+		struct timeval start_time, end_time;
+		gettimeofday(&start_time, NULL);
+
+		do {
+			ret = mbedtls_ssl_read( &ssl, big_buf, sizeof(big_buf) );
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 16]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+			if( ret == MBEDTLS_ERR_SSL_WANT_READ ||
+				ret == MBEDTLS_ERR_SSL_WANT_WRITE ){
+				mbedtls_printf(" read error(%d)\n", ret);
+				continue;
+			}
+			received += ret;
+			mbedtls_printf(" read(%d) progress(%d/%d)\n", ret, received, datasize);
+		} while(datasize - received > 0);
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 17]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+		gettimeofday(&end_time, NULL);
+		int duration = (end_time.tv_sec - start_time.tv_sec) * 1000000 +
+			(end_time.tv_usec - start_time.tv_usec);
+		mbedtls_printf(" Transfer is done %dKB %lfmbps\n", datasize/1024,
+					   (double)datasize/(double)duration/(1024*1024)*1000000);
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: add space between operators

**Comment by pillip8282** [comment]
done

**Comment by sunghan-chang** [comment]
@pillip8282 Please remove old commits like "update README"

File: apps/examples/tls_client/tls_client_main.c [hunk 18]
```diff
@@ -831,7 +841,13 @@ int tls_client_cb(void *args)
...
 				goto usage;
 			}
-		} else {
+		} else if(strcmp(p, "aging") == 0) {
+			opt.aging = atoi(q);
+			if(opt.aging <= 0 && opt.aging > MAX_DATA_SIZE) {
+				goto usage;
+			}
+		}
+		else {
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: Put this line at above.
```
} else {
```

File: apps/examples/tls_client/tls_client_main.c [hunk 19]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
 	 */
 	retry_left = opt.max_resend;
 send_request:
+		if (opt.aging > 0) {
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: Need to check indentation. Let's remove one tab

File: apps/examples/tls_client/tls_client_main.c [hunk 20]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+		mbedtls_printf("   start the aging test (%d)B sendbuf size(%d)B\n", opt.aging, sizeof(buf));
+		int send_left = opt.aging;
+
+		// send buf size
+		unsigned int aging_size = htonl(opt.aging);
+		ret = mbedtls_ssl_write(&ssl, (void *)&aging_size, sizeof(unsigned int));
+		if (ret <= 0 && ret != MBEDTLS_ERR_SSL_WANT_READ &&
+			ret != MBEDTLS_ERR_SSL_WANT_WRITE) {
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: Why do you change a line?

File: apps/examples/tls_client/tls_client_main.c [hunk 21]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+			mbedtls_printf(" failed to send transfer size to the tls server(%x)\n", -ret);
+			goto exit;
+		}
+		
+		while (send_left > 0) {
+		    int datasize = send_left < MBEDTLS_SSL_MAX_CONTENT_LEN? send_left : MBEDTLS_SSL_MAX_CONTENT_LEN;
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: remove space for indentaion.

File: apps/examples/tls_client/tls_client_main.c [hunk 22]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+			for(written = 0, frags = 0; written < datasize; written += ret, frags++ )
+			{
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]:
```
for (xx) {
```

File: apps/examples/tls_client/tls_client_main.c [hunk 23]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+				int bufsize = datasize - written;
+				while((ret = mbedtls_ssl_write(&ssl, buf + written, bufsize ) ) <= 0 )
+				{
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: same as previous

File: apps/examples/tls_client/tls_client_main.c [hunk 24]
```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
...
+					if(ret != MBEDTLS_ERR_SSL_WANT_READ &&
+						ret != MBEDTLS_ERR_SSL_WANT_WRITE )
+					{
```
Comment (sunghan-chang) [inline apps/examples/tls_client/tls_client_main.c]: same

File: apps/examples/tls_server/tls_server_main.c [hunk 25]
```diff
@@ -1238,6 +1244,16 @@ int tls_server_cb(void *args)
 			opt.sni = q;
 		} else if (strcmp(p, "retry") == 0) {
 			opt.retry = atoi(q);
+		} else if (strcmp(p, "aging") == 0){
+			if(strcmp (q, "true") == 0) {
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 26]
```diff
@@ -1238,6 +1244,16 @@ int tls_server_cb(void *args)
 			opt.sni = q;
 		} else if (strcmp(p, "retry") == 0) {
 			opt.retry = atoi(q);
+		} else if (strcmp(p, "aging") == 0){
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 27]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+		unsigned char big_buf[1024];
+		int received = 0;
+
+		// read transfer size from the client
+		unsigned int datasize;
+		ret = mbedtls_ssl_read(&ssl, (void *)&datasize, sizeof(unsigned int));
+
+		if(ret <= 0 &&
+			ret != MBEDTLS_ERR_SSL_WANT_READ &&
+			ret != MBEDTLS_ERR_SSL_WANT_WRITE ){
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

File: apps/examples/tls_server/tls_server_main.c [hunk 28]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+			goto exit;
+		}
+		datasize = ntohl(datasize);
+		mbedtls_printf(" < Read the large data from the client (%d)B\n", datasize);
+
+		struct timeval start_time, end_time;
+		gettimeofday(&start_time, NULL);
+
+		do {
+			ret = mbedtls_ssl_read(&ssl, big_buf, sizeof(big_buf) );
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: remove unnecessary space

File: apps/examples/tls_server/tls_server_main.c [hunk 29]
```diff
@@ -1924,6 +1940,48 @@ int tls_server_cb(void *args)
...
+			if(ret == MBEDTLS_ERR_SSL_WANT_READ ||
```
Comment (jeongchanKim) [inline apps/examples/tls_server/tls_server_main.c]: check space

**Pull request closed without merging**